src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const fullList = document.querySelector('ul');
+const employees = document.querySelectorAll('li');
+
+function convertToNumber(element) {
+  return Number(element.slice(1).replaceAll(',', ''));
+}
+
+function sortList(elements) {
```

Comment:
Same here, it's not a generic function, but a very specific sorting by salary, so why don't we call it sortListBySalary?